docs/docs.trychroma.com/pages/getting-started.md

```diff
@@ -74,32 +76,40 @@ Run the Chroma backend:
 
 {% codetabs customHeader="sh" %}
 {% codetab label="CLI" %}
+
 ```bash {% codetab=true %}
-chroma run --path /getting-started # [!code $]
+chroma run --path ./getting-started # [!code $]
```

Comment:
In my experience, this would previously reference the root of the filesystem so updated this to refer to current dir